watertap/examples/flowsheets/electrodialysis/electrodialysis_1stack_conc_recirc_ui.py

```diff
@@ -0,0 +1,629 @@
+
+    # System costing
+    exports.add(
+        obj=fs.costing.utilization_factor,
+        name="Utilization factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Utilization factor - [annual use hours/total hours in year]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.TIC,
+        name="Total Installed Cost",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=1,
+        description="Total Installed Cost (TIC)",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.TPEC,
+        name="Total Purchased Equipment Cost",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=1,
+        description="Total Purchased Equipment Cost (TPEC)",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.total_investment_factor,
+        name="Total investment factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Total investment factor [investment cost/equipment cost]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.maintenance_labor_chemical_factor,
+        name="Maintenance-labor-chemical factor",
+        ui_units=1 / pyunits.year,
+        display_units="fraction/year",
+        rounding=2,
+        description="Maintenance-labor-chemical factor [fraction of investment cost/year]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.capital_recovery_factor,
+        name="Capital annualization factor",
+        ui_units=1 / pyunits.year,
+        display_units="fraction/year",
+        rounding=2,
+        description="Capital annualization factor [fraction of investment cost/year]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.electricity_cost,
+        name="Electricity cost",
+        ui_units=fs.costing.base_currency / pyunits.kWh,
+        display_units="$/kWh",
+        rounding=3,
+        description="Electricity cost",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+
+    # Feed
+    exports.add(
+        obj=fs.feed.properties[0].flow_vol_phase["Liq"],
+        name="Volumetric flow rate",
+        ui_units=pyunits.m**3 / pyunits.hour,
+        display_units="m3/hr",
+        rounding=2,
+        description="Inlet volumetric flow rate",
+        is_input=False,
+        is_output=True,
+        output_category="Feed",
+    )
```

Comment:
This variable has to be deleted since `fs.feed.properties[0].flow_vol_phase["Liq"]` is already assigned to an object (the first variable exported in this file). To resolve this, you should change `is_output` to True in the prior example and add the output_category name.